src/libraries/System.Net.Security/tests/InternalUnitTests/System.Net.Security.InternalUnit.Tests.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Why is a new project needed here?  The existing UnitTests (not FunctionalTests) project should be usable for this, yes?  It similarly builds product src into the test to validate internals.

Reply:
The whole System.Net.Security UnitTests project is disabled on Android https://github.com/dotnet/runtime/blob/8fbf206d0e518b45ca855832e8bfb391afa85972/src/libraries/tests.proj#L139 , so the MD5 unit test would not run on Android where we are actually going to use it.

However, the bug that it is disabled against is marked as resolved. Does it still need to be disabled?

Reply:
Certainly worth a check.

Reply:
> Why is a new project needed here? The existing UnitTests (not FunctionalTests) project should be usable for this, yes? It similarly builds product src into the test to validate internals.

It's **temporary** extracted to a separate project for simplicity as it's small and can run as part of the limited set of smoke tests in CI and doesn't require the full cycle of the library tests. In the end, the MD4 tests would definitely go to the UnitTests project. I can double-check if the UnitTests project can be enabled for Android.

Reply:
Opened https://github.com/dotnet/runtime/pull/63669 to enable System.Net.Security UnitTest project.

Reply:
Moved the MD4 tests to the System.Net.Security UnitTest project.